java/fundamentals/tips-ii/how-range-operations-work.md

```diff
@@ -1,46 +1,55 @@
 ---
 author: catalin
 
-levels:
-
-  - basic
-
 type: normal
 
 category: must-know
 
 aspects:
-
   - introduction
 
 notes: 'https://insights.enki.com/#/contrib/others/55c83ba83f06ab320051aa8a?search=kha'
 
 ---
 
-# How range operations work
+# How Range Operations Work
 
 ---
 ## Content
 
```

Comment:
Can we add a very short intro paragraph about what a range is in this context? Ranges can mean different things, but here we mean sequences of values bounded by position

"A range can refer to an arbitrary, ordered sequence of values within a list-like value. The boundaries of a range are often represented by indexes."